velox/exec/Window.h

```diff
@@ -148,6 +148,51 @@ class Window : public Operator {
       vector_size_t numRows,
       vector_size_t* rawFrameBounds);
 
+  struct RangeSearchParams {
+    // This is the number of rows incremented when searching the range value.
+    // It is +1 for searching following rows, and -1 for searching preceding
+    // rows. This can be optimized to be > 1 rows also.
```

Comment:
> This can be optimized to be > 1 rows also.

Would you clarify this? Perhaps, update PR description to explain what optimizations you are planning to add in follow-up PRs.